chapter-07/recipe-06/README.md

```diff
@@ -2,4 +2,5 @@
 
 Abstract to be written ...
 
+- [cxx-example](cxx-example/)
 - [example](example/)
```

Comment:
Same comment as above the `cxx-example` was removed, but the empty folder still lies around, confusing `generate-readmes.py`.